wiki/Play_Styles/en.md

```diff
@@ -1,165 +1,169 @@
----
-needs_cleanup: true
----
-<!-- TODO: some of these descriptions are terrible and the whole thing needs a decent review. -->
-
 # Play styles
 
-This page will detail the various different ways in which players can choose to play in their respective game modes.
+This page will detail the various ways in which players can choose to play in their respective game modes.
 
 ## ![](/wiki/shared/mode/osu.png) osu!
 
 ### Mouse-only
 
-Mouse-only was once the _only_ way to play osu! until other playing methods were supported over time.
+Mouse-only was once the *only* way to play osu! until other playing methods were supported over time.
+
+Also known as *pure-mouse*, mouse-only players navigate the in-game cursor with a mouse as well as tap beats, follow sliders, and spin spinners by only clicking the left/right mouse buttons.
```

Comment:
```suggestion
Also known as *pure-mouse*, mouse-only players navigate the in-game cursor with a mouse while tapping beats, following sliders, and spinning spinners with only clicking the left and right mouse buttons.
```

Reply:
Why remove the "and" and replce "by" with "with"?